### PR TITLE
Cloud changelog - Oct 6th

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,55 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a new query parameter to include deleted posts as long as it's requested by a System Admin in ``/api/v4/channels/{channel_id}/posts``.
+
+### Bug Fixes
+ - Fixed an issue with the badge count on the mobile app when a channel/thread was viewed.
+ - Fixed an issue where typing ``@`` in the right-hand side rendered a cut-off user suggestion list.
+ - Fixed an issue where an error screen was briefly flashed when the first Admin signed up into a new server.
+ - Fixed an issue where users were unable to add Japanese comments correctly in the message **Forward** modal.
+ - Fixed an issue where unsaved edits to a post were lost when switching channels or threads.
+ - Fixed an issue on larger screen sizes where the Insights widgets were pushed to the side when the right-hand side was open.
+ - Fixed an issue where the ability to forward messages from public channels to someone whom you've never Direct Messaged before wasn’t possible.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:45]:
```suggestion
 - Fixed an issue where the ability to forward messages from public channels wasn't possible when messaging someone directly for the first time.
```

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,55 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where custom emojis were sometimes not visible in **Insights > Top Reactions**.
+ - Fixed an issue where channels with no posts in a particular time window didn't show in **Insights > Least Active Channel**.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:47]:
```suggestion
 - Fixed an issue where channels with no posts within a particular timeframe didn't show in **Insights > Least Active Channel**.
```

**Review by agnivade — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,55 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+#### User Interface (UI)
+ - Added Insights to the channel switcher.
+ - Added a button to easily copy the content of text or code files in file previews.
+ - The team unread icon for muted channels is now hidden in the sidebar.
+ - Updated the designs for the “payment failed” email notifications.
+
+#### Administration
+ - A ``batchSize`` option has been added to the mattermost export CLI command to limit the number of items exported. By default, if it is not included, it exports all the posts.
+ - Added a credit card update page that communicates arrears for Admins.
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/install/cloud-changelog.md:26]: Adding @nickmisasi to help clarify.

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+### Improvements
+
+#### User Interface (UI)
+ - Added Insights to the channel switcher.
+ - Added a button to easily copy the content of text or code files in file previews.
+ - The team unread icon for muted channels is now hidden in the sidebar.
+ - Updated the designs for the “payment failed” email notifications.
+
+#### Administration
+ - A ``batchSize`` option has been added to the ``mattermost export`` CLI command to limit the number of items exported. By default, if it is not included, it exports all posts.
```
Comment (spirosoik) [inline source/install/cloud-changelog.md:25]: do we mean `mmctl` export?

**Review by spirosoik — CHANGES_REQUESTED** [review]
LGTM, just a small comment for validation

File: source/install/cloud-changelog.md (hunk 4 above)
Comment (cwarnermm) [reply source/install/cloud-changelog.md:25]: If we are referring to the CLI command, docs updates for v7.4 will be available here: https://docs.mattermost.com/manage/command-line-tools.html#mattermost-export

Comment (amyblais) [reply source/install/cloud-changelog.md:25]: This is referring to the CLI command.

**Review by spirosoik — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Pull request merged**